### PR TITLE
402 pdf links and date format

### PR DESCRIPTION
**Pull request by sadlerw**
* pdf links
* date format
screen shot 2019-01-14 at 5 10 44 pm
 is wrong? corrected to:
screen shot 2019-01-14 at 5 16 39 pm

### CODE REVIEW

File: web-client/src/views/CaseDetailPetitioner.jsx [hunk 1]
```diff
@@ -182,7 +182,7 @@ export default connect(
                     <tr key={idx}>
                       <td className="responsive-title">
                         <span className="responsive-label">Activity date</span>
-                        {moment(document.createdAt).format('l')}
+                        {moment(document.createdAt).format('L')}
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailPetitioner.jsx]: Did we say these should also probably be computed?

Comment (sadlerw) [reply web-client/src/views/CaseDetailPetitioner.jsx]: they are being refactored in a later story - this entire view is being changed

Comment (ericsorenson) [inline web-client/src/views/CaseDetailPetitioner.jsx]: This should be using the computed value

Comment (sadlerw) [reply web-client/src/views/CaseDetailPetitioner.jsx]: As I already stated the entire file needs to be refactored to the computed.  That will be handled in another story that is coming up soon.

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**